use_case.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+printf "show movies after given dates\n"
+curl -H "Content-Type:application/json" -X GET 'http://localhost:8080/movies/byScreeningTimeResponse?startDateTime=2021-11-22T10:20&endDateTime=2021-11-22T12:20'
```

Comment:
to jest nieprawda - Ty nie wysyłasz body więc `Content-Type` to powinien być `Accept`